content/applications/websites/ecommerce/checkout_payment_shipping/shipping.rst

```diff
@@ -0,0 +1,71 @@
+========
+Shipping
+========
+
+Depending on your shipping strategy, you have the choice to either use your :ref:`own shipping
+methods <ecommerce-own-shipping>`, or use an integration with an :ref:`existing shipping provider
+<ecommerce-shipping-providers>`.
+
+Own shipping methods
+====================
+
+.. _ecommerce-own-shipping:
+
+You can create your own custom shipping methods and define rules to compute shipping costs. To do
+so, go to :menuselection:`Website --> Configuration --> Shipping Methods`, and either select an
+**existing** shipping method, or :guilabel:`Create` one. When creating a shipping method, you can
+choose between
+:doc:`Fixed Price </applications/inventory_and_mrp/inventory/shipping/setup/delivery_method>`,
+:doc:`Based on Rules </applications/inventory_and_mrp/inventory/shipping/setup/delivery_method>`,
+and :guilabel:`Pickup in store`.
+
+Pickup in store
+---------------
```

Comment:
Do you explain this here only because it's missing from the other page (which seems quite deprecated)?

If it's specific to eCommerce, then it's fine like this. If not, I advise you either add the missing info to the other inventory page, even if the other page is deprecated, or contact the content responsible of this other page to see what can be done.

Reply:
This is website-specific (This only talks about how to enable it within and configure the basics through the eCommerce app), it doesn't go into any details about configuration of "On Site Pick-up" nor its settings.